crates/client/eth/Cargo.toml

```diff
@@ -0,0 +1,70 @@
+[package]
+authors.workspace = true
+description = "This crate is responsible to handle l1 communication"
+edition = "2021"
+homepage = "https://github.com/KasarLabs/deoxys"
+license = "MIT"
+name = "dc-eth"
+publish = false
+repository = "https://github.com/KasarLabs/deoxys"
+version = "0.1.0"
+
+[features]
+default = []
+m = ["dep:rodio"]
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
```

Comment:
I think most of those dependencies are not needed for that crate

Reply:
fixed